<launch>
    <node pkg="logger" type="logger_node" name="logger_node" output="screen">
        <rosparam command="load" file="$(find logger)/config/params.yaml"/>
    </node>
</launch>